test/integration/basic.ts

```diff
@@ -94,7 +94,7 @@ test('it should be converted on success', (done) => {
     })
     .then((response) => {
       expect(JSON.stringify(response.data)).toBe(JSON.stringify(camelData));
-      expect(response.headers.contentType).toBe('application/json');
+      expect(response.headers.contentType || response.headers['Content-Type']).toBe('application/json');
```

Comment:
What's the purpose of these changes?

Reply:
@mpyw `response.headers.contentType` is no longer available in axios >= 1.0, so I added `response.headers['Content-Type']` which actually exists. To keep backward compatibility, I left the previous one with logical OR. This mean this tests can be run even with axios < 1.0.

Reply:
@siketyan 

Was `headers.contentType` really generated by `axios`, not by `axios-case-converter`? I suspect that the camelCase conversion no longer works for latest `axios`.

Reply:
[axios v1.x bug: response header mutations via `transformResponse` are discarded · Issue #47 · mpyw/axios-case-converter](https://github.com/mpyw/axios-case-converter/issues/47)